sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,9 @@
+select "g"."name",
+ count(*) as "genres"
+
+from "films" as "f"
+join "filmGenre" as "fg" using ("filmId")
+join "genres" as "g" using ("genreId")
+join "castMembers" as "c" using ("filmId")
+where "actorId" = 178
```

Comment:
For this to be robust, you can't make the assumption that `'Lisa Monroe'` will have the `actorId` of `178`.

---

![image](https://user-images.githubusercontent.com/16143771/151600925-49df3cff-2df6-46af-b9b6-90148c8b3659.png)